src/overlays/actors/ovl_Obj_Kibako2/z_obj_kibako2.c

```diff
@@ -65,7 +65,7 @@ s32 ObjKibako2_ContainsSkulltula(ObjKibako2* this, PlayState* play) {
     if ((u16)actorSpawnParam & 3) {
         flag = ((actorSpawnParam & 0x3FC) >> 2) & 0xFF;
     }
-    return !(flag >= 0 && Flags_GetTreasure(play, flag));
+    return !((flag >= 0) && Flags_GetTreasure(play, flag));
```

Comment:
I assume this doesn't match, right?
```suggestion
    return (flag < 0) || !Flags_GetTreasure(play, flag);
```

Reply:
Unfortunately not :(